csharp/native/SubTasking/Client/Program.cs

```diff
@@ -0,0 +1,216 @@
+
+      // Create client for task submission
+      var taskClient = new Tasks.TasksClient(channel);
+
+      // Create client for result creation
+      var resultClient = new Results.ResultsClient(channel);
+
+      // Create client for session creation
+      var sessionClient = new Sessions.SessionsClient(channel);
+
+      // Create client for events listening
+      var eventClient = new Events.EventsClient(channel);
+
+      // Default task options that will be used by each task if not overwritten when submitting tasks
+      var taskOptions = new TaskOptions
+                        {
+                          MaxDuration = Duration.FromTimeSpan(TimeSpan.FromHours(1)),
+                          MaxRetries  = 2,
+                          Priority    = 1,
+                          PartitionId = partition,
+                          Options =
+                          {
+                            new MapField<string, string>
+                            {
+                              {
+                                "UseCase", "Launch"
+                              },
+                            },
+                          },
+                        };
+
+      // Request for session creation with default task options and allowed partitions for the session
+      var createSessionReply = sessionClient.CreateSession(new CreateSessionRequest
+                                                           {
+                                                             DefaultTaskOption = taskOptions,
+                                                             PartitionIds =
+                                                             {
+                                                               partition,
+                                                             },
+                                                           });
+
+      Console.WriteLine($"sessionId: {createSessionReply.SessionId}");
+
+      // Create the result metadata and keep the id for task submission
+      var resultId = resultClient.CreateResultsMetaData(new CreateResultsMetaDataRequest
+                                                        {
+                                                          SessionId = createSessionReply.SessionId,
+                                                          Results =
+                                                          {
+                                                            new CreateResultsMetaDataRequest.Types.ResultCreate
+                                                            {
+                                                              Name = "Result",
+                                                            },
+                                                          },
+                                                        })
+                                 .Results.Single()
+                                 .ResultId;
```

Comment:
Can you also print this result since it is the one we need for the application to complete ?